Common/Settings.ini

```diff
@@ -274,6 +274,9 @@ WhiteShaderFix = 1
 ; Fixes spawning placements for objects in Woodside Apartments Room 205. (0|1)
 WoodsideRoom205Fix = 1
 
+; Restores specularity to characters and objects throughout the game. (0|1)
+SpecularFix = 1
```

Comment:
For the most part, in each section of the ini we alphabetize the items.

Reply:
Done